patches/minecraft/net/minecraft/server/management/PlayerList.java.patch

```diff
@@ -169,23 +175,29 @@
          this.func_72375_a(p_187242_1_, worldserver);
          p_187242_1_.field_71135_a.func_147364_a(p_187242_1_.field_70165_t, p_187242_1_.field_70163_u, p_187242_1_.field_70161_v, p_187242_1_.field_70177_z, p_187242_1_.field_70125_A);
          p_187242_1_.field_71134_c.func_73080_a(worldserver1);
-@@ -576,17 +626,27 @@
+@@ -576,17 +632,33 @@
          {
              p_187242_1_.field_71135_a.func_147359_a(new SPacketEntityEffect(p_187242_1_.func_145782_y(), potioneffect));
          }
 +        net.minecraftforge.fml.common.FMLCommonHandler.instance().firePlayerChangedDimensionEvent(p_187242_1_, i, p_187242_2_);
      }
  
++    @SuppressWarnings("unused")
      public void func_82448_a(Entity p_82448_1_, int p_82448_2_, WorldServer p_82448_3_, WorldServer p_82448_4_)
      {
 -        double d0 = p_82448_1_.field_70165_t;
 -        double d1 = p_82448_1_.field_70161_v;
 +        transferEntityToWorld(p_82448_1_, p_82448_2_, p_82448_3_, p_82448_4_, p_82448_4_.func_85176_s());
 +    }
 +
-+    @SuppressWarnings("unused")
++    // TODO: Remove (1.13)
 +    public void transferEntityToWorld(Entity p_82448_1_, int p_82448_2_, WorldServer p_82448_3_, WorldServer p_82448_4_, net.minecraft.world.Teleporter teleporter)
```

Comment:
You can safely remove this since all `Teleporter` instances are going to be `ITeleporter` instances.

Reply:
Not really, I think compiled code may still hurt.

Reply:
See my previous reply regarding that, the JVM doesn't care about instance of when invoking the method, all it cares about is signature.